docs/docs/examples/vault.md

```diff
@@ -0,0 +1,264 @@
+:::
+</TabItem>
+<TabItem value="k3s-qemu-tdx" label="Bare metal (TDX)">
+```sh
+contrast generate --reference-values k3s-qemu-tdx deployment/
+```
+:::note[Missing TCB values]
+On bare-metal TDX, `contrast generate` is unable to fill in the `MinimumTeeTcbSvn` and `MrSeam` TCB values as they can vary between platforms.
+They will have to be filled in manually.
+If you don't know the correct values use `ffffffffffffffffffffffffffffffff` and `000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000` respectively and observe the real values in the error messages in the following steps. This should only be done in a secure environment.
+:::
+</TabItem>
+</Tabs>
+
+:::note[Runtime class and Initializer]
+
+The deployment YAML shipped for this demo is already configured to be used with Contrast.
+A [runtime class](../components/runtime) `contrast-cc`
+was added to the pods to signal they should be run as Confidential Containers. During the generation process,
+the Contrast [Initializer](../components/overview.md#the-initializer) will be added as an init container to these
+workloads. It will attest the pod to the Coordinator and fetch the workload certificates and the workload secret.
+
+Further, the deployment YAML is also configured with the Contrast [service mesh](../components/service-mesh.md).
+The configured service mesh proxy provides transparent protection for the communication between
+the Vault server and client.
+:::
+
+### Deploy the Coordinator
+
+Deploy the Coordinator resource first by applying its resource definition:
+
+```sh
+kubectl apply -f deployment/coordinator.yml
+```
+
+### Set the manifest
+
+Configure the Coordinator with a manifest. It might take up to a few minutes
+for the load balancer to be created and the Coordinator being available.
+
+```sh
+coordinator=$(kubectl get svc coordinator -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
+echo "The user API of your Contrast Coordinator is available at $coordinator:1313"
+contrast set -c "${coordinator}:1313" deployment/
+```
+
+The CLI will use the reference values from the manifest to attest the Coordinator deployment
+during the TLS handshake. If the connection succeeds, it's ensured that the Coordinator
+deployment hasn't been tampered with.
+
+### Deploy Vault
+
+Now that the Coordinator has a manifest set, which defines the Vault deployment as an allowed workload,
+we can deploy the application:
+
+```sh
+kubectl apply -f deployment/
+```
+:::note[Persistent workload secrets]
+
+During the initialization process of the workload pod, the Contrast Initializer
+sends an attestation report to the Coordinator and receives a workload secret
+derived from the Coordinator's secret seed and the `workloadSecretID` specified in the
+manifest, and writes it to a secure in-memory `volumeMount`.
+
+:::
```

Comment:
```suggestion

```
I don't think this note is relevant here - this page is not so much about workload secrets, and we're not even using the workload secret anywhere.